docs/test-pages/TCD-009.adoc

```diff
@@ -0,0 +1,46 @@
+= TCD-009
+
+== Purpose
+
+Verifying the primitive types and their working. This case is verifying the optional primitive fields by adding values. This is a simple happy path.
+
+[NOTE]
+This is a reverse engineered test case.
+
+=== Tested requirements
+
+REQ-TYPE-001
+
+=== Used JSL model
+
+xref:resources/Primitives.jsl[Primitives.jsl]
+
+== Test scenario
+
+. Create and save MyEntityWithOptionalFields with these attributes:
+    * integerAttr = 1;
+    * scaledAttr = 1.23;
+    * stringAttr = "test";
+    * regexAttr = "+36 333-333-3333";
+    * boolAttr = true;
+    * dateAttr = "2022.07.11";
+    * timestampAttr = "2022-07-11T19:09:33Z";
+    * timeAttr = "23:59:59";
+    * binaryAttr = File(xref:resources/test.txt[test.txt]);
+    * enumAttr = MyEnum#Bombastic;
+
+. Read from the database the instance of MyEntityWithOptionalFields that saved in the previous step (entity1).
```

Comment:
"Re-read the instance of MyEntityWithOptionalFields that has been saved in the previous step (entity1)."